models/transaction_test.go

```diff
@@ -473,7 +473,7 @@ func TestValidateConsensusRules(t *testing.T) {
 		// All those fields are summed up to at most 213 bytes.
 		baseSize := uint64(213)
 		dataSize := TxMaxSize - baseSize
-		gasLimit := uint64(2_500_000)
+		gasLimit := uint64(5_500_000)
```

Comment:
Change comes from: `EIP-7623: Increase calldata cost`.
See: https://mixbytes.io/blog/the-prague-electra-pectra-hardfork-explained#rec860520137 .